TreeMaker/python/makeJetVars.py

```diff
@@ -424,30 +424,51 @@ def makeJetVarsAK8(self, process, JetTag, suff, storeProperties, SkipTag=cms.VIn
+            JetPropertiesAK8.pfDeepBoostedDiscriminatorsJetTagsTvsQCD = cms.vstring("pfDeepBoostedDiscriminatorsJetTags:TvsQCD")
+            JetPropertiesAK8.pfDeepBoostedDiscriminatorsJetTagsWvsQCD = cms.vstring("pfDeepBoostedDiscriminatorsJetTags:WvsQCD")
+            JetPropertiesAK8.pfDeepBoostedDiscriminatorsJetTagsZvsQCD = cms.vstring("pfDeepBoostedDiscriminatorsJetTags:ZvsQCD")
+            JetPropertiesAK8.pfDeepBoostedDiscriminatorsJetTagsHbbvsQCD = cms.vstring("pfDeepBoostedDiscriminatorsJetTags:HbbvsQCD")
+            JetPropertiesAK8.pfDeepBoostedDiscriminatorsJetTagsZHbbvsQCD = cms.vstring("pfDeepBoostedDiscriminatorsJetTags:ZHbbvsQCD")
+            JetPropertiesAK8.pfDeepBoostedDiscriminatorsJetTagsZbbvsQCD = cms.vstring("pfDeepBoostedDiscriminatorsJetTags:ZbbvsQCD")
+            JetPropertiesAK8.pfDeepBoostedDiscriminatorsJetTagsbbvsLight = cms.vstring("pfDeepBoostedDiscriminatorsJetTags:bbvsLight")
+            JetPropertiesAK8.pfMassDecorrelatedDeepBoostedDiscriminatorsJetTagsTvsQCD = cms.vstring("pfMassDecorrelatedDeepBoostedDiscriminatorsJetTags:TvsQCD")
+            JetPropertiesAK8.pfMassDecorrelatedDeepBoostedDiscriminatorsJetTagsWvsQCD = cms.vstring("pfMassDecorrelatedDeepBoostedDiscriminatorsJetTags:WvsQCD")
+            JetPropertiesAK8.pfMassDecorrelatedDeepBoostedDiscriminatorsJetTagsZvsQCD = cms.vstring("pfMassDecorrelatedDeepBoostedDiscriminatorsJetTags:ZvsQCD")
+            JetPropertiesAK8.pfMassDecorrelatedDeepBoostedDiscriminatorsJetTagsHbbvsQCD = cms.vstring("pfMassDecorrelatedDeepBoostedDiscriminatorsJetTags:HbbvsQCD")
+            JetPropertiesAK8.pfMassDecorrelatedDeepBoostedDiscriminatorsJetTagsZHbbvsQCD = cms.vstring("pfMassDecorrelatedDeepBoostedDiscriminatorsJetTags:ZHbbvsQCD")
+            JetPropertiesAK8.pfMassDecorrelatedDeepBoostedDiscriminatorsJetTagsZbbvsQCD = cms.vstring("pfMassDecorrelatedDeepBoostedDiscriminatorsJetTags:ZbbvsQCD")
+            JetPropertiesAK8.pfMassDecorrelatedDeepBoostedDiscriminatorsJetTagsbbvsLight = cms.vstring("pfMassDecorrelatedDeepBoostedDiscriminatorsJetTags:bbvsLight")
             self.VectorDouble.extend([
-                'JetProperties'+suff+':tDiscriminatorDeep(Jets'+suff+'_tDiscriminatorDeep)',
-                'JetProperties'+suff+':wDiscriminatorDeep(Jets'+suff+'_wDiscriminatorDeep)',
-                'JetProperties'+suff+':zDiscriminatorDeep(Jets'+suff+'_zDiscriminatorDeep)',
-                'JetProperties'+suff+':hDiscriminatorDeep(Jets'+suff+'_hDiscriminatorDeep)',
-                'JetProperties'+suff+':tDiscriminatorDeepDecorrel(Jets'+suff+'_tDiscriminatorDeepDecorrel)',
-                'JetProperties'+suff+':wDiscriminatorDeepDecorrel(Jets'+suff+'_wDiscriminatorDeepDecorrel)',
-                'JetProperties'+suff+':zhDiscriminatorDeepDecorrel(Jets'+suff+'_zhDiscriminatorDeepDecorrel)',
+                'JetProperties'+suff+':pfDeepBoostedDiscriminatorsJetTagsTvsQCD(Jets'+suff+'_pfDeepBoostedDiscriminatorsJetTagsTvsQCD)',

```

Comment:
I'm going to complain yet again about way too long branch names... "pfDeepBoostedDiscriminatorsJetTags" is just unnecessary. Proposals:
* DeepDiscr[iminator]TvsQCD and DeepDecorrelDiscrTvsQCD
* DeepTagTvsQCD and DeepDecorrelTagTvsQCD

Reply:
Okay, I changed the names per your recommendation.